assets/sass/config/_variables.scss

```diff
@@ -65,6 +65,9 @@ $c-pagespeed-fast: $c-green-700;
 $c-pagespeed-average: $c-yellow-900;
 $c-pagespeed-slow: $c-red-600;
 
+// Idea Hub colors
+$c-idea-hub-orange: #f78f01;
```

Comment:
@johnPhillips Let's move the variable a bit further down in the file where we have other `$c-idea-hub-*` variables. :)

Reply:
@asvinb maybe those colors below should move up here instead since PSI colors are above? Or maybe all the module colors should move to the bottom? 😄

Reply:
@aaemnnosttv Actually it's good you caught this. Ideally we should define the colors first (without any specific module prefix to it) using https://chir.ag/projects/name-that-color/
In this case it'll be : 
```
$c-pizazz: #f78f01;
```

Then we use the `$c-pizazz` variable for `$c-idea-hub-orange`

Reply:
Trying to summarize. @aaemnnosttv and @asvinb Are you good with this approach?
```diff
... 

 $c-shark: #202124;
 $c-oslo-gray: #80868b;
+ $c-pizazz: #f78f01;  // 1. Define new color with the others at the top

 // PageSpeed Insights colors
 $c-pagespeed-fast: $c-green-700;
 $c-pagespeed-average: $c-yellow-900;
 $c-pagespeed-slow: $c-red-600;

- // Idea Hub colors
- $c-idea-hub-orange: #f78f01;

 $c-base: $c-white;
 $c-primary: $c-boulder;

...

 $c-user-input-placeholder: $c-heather;
 $c-user-input-color: $c-denim;

+ // Idea Hub colors
+ $c-idea-hub-orange: $c-pizzaz;  // 2. Move the $c-idea-hub-orange declaration to the bottom with the other Idea Hub vars
 $c-idea-hub-background: $c-white;
 $c-idea-hub-background-hover: $c-gallery-2;
 $c-idea-hub-button-hover: $c-iron;
```

Reply:
Works for me @johnPhillips 😁

Reply:
LGTM @johnPhillips